internal/kibana/client.go

```diff
@@ -81,22 +83,26 @@ func CertificateAuthority(certificateAuthority string) ClientOption {
 }
 
 func (c *Client) get(resourcePath string) (int, []byte, error) {
-	return c.sendRequest(http.MethodGet, resourcePath, nil)
+	return c.sendRequest(http.MethodGet, resourcePath, nil, map[string]string{"content-type": defaultContentType})
 }
 
 func (c *Client) post(resourcePath string, body []byte) (int, []byte, error) {
-	return c.sendRequest(http.MethodPost, resourcePath, body)
+	return c.postWithContentType(resourcePath, defaultContentType, body)
+}
+
+func (c *Client) postWithContentType(resourcePath, contentTypeHeader string, body []byte) (int, []byte, error) {
+	return c.sendRequest(http.MethodPost, resourcePath, body, map[string]string{"content-type": contentTypeHeader})
```

Comment:
Any other suggestion to add here the custom content-type header ? @jsoriano 
In case of uploading/installing a zip package that header must be `application/zip`. This header is set [here](https://github.com/elastic/elastic-package/pull/1160/files#diff-5957946cb38e2c6b7a662962ed2a0de756f43e76794e775114ab48f9b65a0e65R44)